enterprise/admin/forms.py

```diff
@@ -35,6 +37,10 @@ class ManageLearnersForm(forms.Form):
         label=_("Also enroll these learners in this course"), required=False,
         help_text=_("Provide a course ID if enrollment is desired."),
     )
+    program = forms.CharField(
+        label=_("Or enroll these learners in this program"), required=False,
```

Comment:
Is this either/or selection (I guess it's actually Also/Or) enforced in the code?